### PR TITLE
Liberty related fixes (Non-Neutron)

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: chef/cookbooks/glance/recipes/api.rb [hunk 1]
```diff
@@ -98,6 +98,14 @@
 glance_stores = node.default[:glance][:glance_stores]
 glance_stores += ["glance.store.vmware_datastore.Store"] unless node[:glance][:vsphere][:host].empty?
 
+# glance_stores unconditionally enables cinder and swift stores, so we need
+# to install the necessary clients
+case node[:platform_family]
+when "suse"
+  package "python-cinderclient"
+  package "python-swiftclient"
```
Comment (vuntz) [inline chef/cookbooks/glance/recipes/api.rb]: Shouldn't they be deps of the package instead?

Comment (dirkmueller) [inline chef/cookbooks/glance/recipes/api.rb]: well, we explicitely enable those stores, by default only those are enabled that are available. Thats why I thought a requires in the package wouldn't make a lot of sense.

Comment (vuntz) [inline chef/cookbooks/glance/recipes/api.rb]: Hrm, okay. Any reason to make this specific to the `suse` family? :-)

Comment (dirkmueller) [inline chef/cookbooks/glance/recipes/api.rb]: didn't want to figure out the package names for non-suse.. :) but we can remove the check

**Comment by vuntz** [comment]
+1

**Comment by tboerger** [comment]
LGTM

**Comment by toabctl** [comment]
+1

**Pull request merged**